### ISSUE TITLE
Listing customer_account_id when we don't know it

### ISSUE THREAD

**Issue by mledwards** [Opteo/google-ads-api#74]
Hi there,

I'm using oAuth so my customers can provide access to their Google Ads accounts, which is working fine to get the refresh token for the `Customer` function, but I can't find a way to list the `customer_account_id` without them specifying it manually.

```
customer.Customer({
        refresh_token: GOTTHIS,
        customer_account_id: NOTGOTTHIS
});

Any ideas?

**Comment by avermeil** [comment]
Hey -- sorry about leaving this issue hanging.

This isn't something we support at the moment -- We haven't yet put much thought into MCC/customer-centric operations such as CID discovery. We will eventually, but it's not a priority for us right now. Would you consider a PR to propose a solution?

**Comment by mledwards** [comment]
I was thinking, it might be something [google-ads-node](https://github.com/opteo/google-ads-node) can handle, as there's this way of authenticating? https://github.com/opteo/google-ads-node#2-token-generation-and-refresh-handling

If not, I'll have a think about a solution.

**Comment by meip** [comment]
We do a simple HTTP-request. Following a small snippet out of my head with `axios`:

```JavaScript
    const accessToken = '<your-access-token>';
    const developerToken = '<your-developer-token>';
    const url = 'https://googleads.googleapis.com/v2/customers:listAccessibleCustomers';
    const headers = {
        Authorization: 'Bearer ' + accessToken,
        'developer-token': developerToken
    };
    return axios
        .get(url, {
            headers
        })
        .then(r => r.data);
```

**Comment by mledwards** [comment]
That's useful, thanks! I'll give it a try.

**Comment by kritzware** [comment]
This is now available to use in google-ads-api@3.7.0 🎉

**Comment by mledwards** [comment]
This is great timing, just about to use it on a project! Thank you @kritzware. <3
